README.md

```diff
@@ -1,10 +1,33 @@
+# NimbleTemplate
+
+[![Test template](https://github.com/nimblehq/elixir-templates/actions/workflows/test_template.yml/badge.svg?branch=develop)](https://github.com/nimblehq/elixir-templates/actions/workflows/test_template.yml)
+
+
+> A Library for initialize new Phoenix projects
+
 ## Introduction
 
-Phoenix/Mix template for projects at [Nimble](https://nimblehq.co/).
+NimbleTemplate is a library for initializing new Phoenix projects at [Nimble](https://nimblehq.co/). 
+It saves us a lot of time on the installation and configuration of new Phoenix projects.
+
+It supports for linting, testing, and much more, so developers can focus on core feature of the application. 
+
+## Features
+
+Supporting libraries:
```

Comment:
should we call it as `addons` instead of libraries 🤔 the with addon we could add Docker, ...